### PR TITLE
home page slogan developed

### PR DESCRIPTION
**Pull request by KavyaReddy732**
# Description
home page slogan with the button which route to signup page
Please provide a short summary explaining what this PR is about.
not working in storybook because there is no proptype in signup component.
homepage
Fixes https://github.com/HackYourFuture-CPH/rate-my-cv/issues/102

# How to test?
add this component in container/homepage.js 
Please provide a short summary how your changes can be tested?

# Checklist

- [ x] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by pdsorensen** [comment]
Let's have @arya56  review this one first.

File: src/client/components/Home/RegisterButton.style.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  width: 100vw;
+  text-align: center;
+  align-items: center;
+  color: white;
+  padding-top: 110px;
+  z-index: 2;
+}
+
+.homepage-slogan:nth-of-type(1){
+  background-color:  #6236FF;
```
Comment (arya56) [inline src/client/components/Home/RegisterButton.style.css:12]: The bgcolor should be consider as this rgba(98, 54, 255, 0.63)

File: src/client/components/Home/RegisterButton.style.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  background-image: url("../../assets/images/portraits/homeimage.png");
```
Comment (arya56) [inline src/client/components/Home/RegisterButton.style.css:13]: Would be nice if image cover all width.

File: src/client/components/Home/RegisterButton.style.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
+.homepage-slogan{
+  height: 63vh;
+  width: 100vw;
+  text-align: center;
+  align-items: center;
+  color: white;
+  padding-top: 110px;
+  z-index: 2;
+}
```
Comment (arya56) [inline src/client/components/Home/RegisterButton.style.css:9]: We need font appropriate font size for h2 and p

File: src/client/components/Home/RegisterButton.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+    const redirect = () => {
+      history.push('/sign-up')
+    }
+    return (
+        <div className="homepage">
+          <section className="homepage-slogan">
+          <h2>Improve Your Resume To Present Your Best Self</h2>  
+          <p>Upload your Resume to get an evaluation</p>
+          <Button buttonName='Register/Sign Up' style={{ backgroundColor: 'orange' }} onClick={redirect}/>
```
Comment (arya56) [inline src/client/components/Home/RegisterButton.js:17]: This color for button :#F25B06 and reverse in order of Register/Sign Up

Comment (arya56) [reply src/client/components/Home/RegisterButton.js:17]: I think the position for the h2 and p also should be added ,, please check figma

File: src/client/components/Home/RegisterButton.style.css [hunk 5]
```diff
@@ -0,0 +1,18 @@
+.homepage-slogan{
+  height: 63vh;
```
Comment (arya56) [inline src/client/components/Home/RegisterButton.style.css:2]: Height needed to be 993px,  around 83vh

File: src/client/components/Home/RegisterButton.style.css [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  width: 100vw;
+  text-align: center;
+  align-items: center;
+  color: white;
+  padding-top: 110px;
+  z-index: 2;
+}
+
+.homepage-slogan:nth-of-type(1){
```
Comment (arya56) [inline src/client/components/Home/RegisterButton.style.css:11]: Wouldn't be better to consider the 63% opacity as well?

**Comment by pdsorensen** [comment]
I would like you to find another name than `registerButton`. Something a bit more specific, e.g `RegisterPage` or maybe someting even better. Cheers!

File: src/client/components/Home/RegisterButton.style.css (hunk 1 above)
Comment (KavyaReddy732) [reply src/client/components/Home/RegisterButton.style.css:12]: this bgcolor  is mentioned in issue #102

File: src/client/components/Home/RegisterButton.style.css (hunk 2 above)
Comment (KavyaReddy732) [reply src/client/components/Home/RegisterButton.style.css:13]: the original image size is very big.if i center the position it's not even displaying the laptop in the picture.so, i cropped the actual image and managed to get this picture.

File: src/client/components/Home/RegisterButton.style.css (hunk 3 above)
Comment (KavyaReddy732) [reply src/client/components/Home/RegisterButton.style.css:9]: they didn't mentioned any font-size, any suggestions??

File: src/client/components/Home/RegisterButton.style.css (hunk 6 above)
Comment (KavyaReddy732) [reply src/client/components/Home/RegisterButton.style.css:11]: if i give the opacity 63 it's very light and it's not looking good as in figma.
home1

**Comment by KavyaReddy732** [comment]
@pdsorensen 
we can just use SignUp maybe

File: src/client/components/Home/RegisterButton.style.css (hunk 6 above)
Comment (KavyaReddy732) [reply src/client/components/Home/RegisterButton.style.css:25]: added the opacity of 75%

**Comment by pdsorensen** [comment]
@KavyaReddy732 i mean for the components filename. Can't we call it something else?  `RegisterButton` dosen't really reflect the other contents that is in the component. 

image

**Review by bytewiz — APPROVED** [review]
seems alright 👍

**Review by pdsorensen — APPROVED** [review]

**Pull request merged**